incubator/istio/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio
+data:
```

Comment:
Talked to Lachie and this is a hard requirement of Istio: it's restricted to one install per namespace.